Source/Mosa.Korlib/System/String.cs

```diff
@@ -885,5 +885,64 @@ private string CreateTrimmedString(int start, int end)
 				return string.Ctor(this, start, len);
 			}
 		}
+
+		public unsafe static string FromPointer(byte* p, int size)
```

Comment:
Use this supported coding pattern instead:

string result = InternalAllocateString(count);
char* chars = result.first_char;

See existing examples in String.cs

Reply:
Alright, I'll do that.